lib/mayaUsd/ufe/UsdHierarchy.cpp

```diff
@@ -216,11 +220,11 @@ Ufe::SceneItem::Ptr UsdHierarchy::defaultParent() const
 {
     // Default parent for USD nodes is the pseudo-root of their stage, which is
     // represented by the proxy shape.
-    auto path = fItem->path();
+    Ufe::Path ufePath = stripInstanceIndexFromUfePath(path());
```

Comment:
If we're going to pop off the whole tail segment, no point in calling stripInstanceIndexFromUfePath().

Reply:
Yup, good catch. Fixed!

Reply:
Reverted these changes in rebased commit 1ced6783c1524a5a9fa82446bd33193642641e87.